packages/tx/src/eip4844Transaction.ts

```diff
@@ -97,7 +97,11 @@ export class BlobEIP4844Transaction extends BaseTransaction<BlobEIP4844Transacti
     if (this.common.isActivatedEIP(1559) === false) {
       throw new Error('EIP-1559 not enabled on Common')
     }
-    this.activeCapabilities = this.activeCapabilities.concat([1559, 2718, 2930, 4844])
+
+    if (this.common.isActivatedEIP(4844) === false) {
+      throw new Error('EIP-4844 not enabled on Common')
```

Comment:
Short super-nit: I think we *always* write "in library x" in all other cases I have read? 🤔 (sorry for this comment in advance)

Reply:
I guess I"m not sure what this error message should look like then.  I don't see anything anywhere in the monorepo in our Typescript code that uses the phrase "in library" so not sure how to adjust this one.  I'm just mirroring the errors through in 1559 checks when 1559 isn't activated but expected to me.

Reply:
Ah, just wanted to suggest to write "enabled in Common" instead of "enabled on Common". Sorry, sorry. 😜 One just shouldn't writing down "short super-nits" at all.